src/pkcs11/interface.rs

```diff
@@ -14,12 +14,16 @@ pub struct CK_INTERFACE {
     pub flags: CK_FLAGS,
 }
 
-pub const KRYATTR_OFFSET: CK_ULONG = 485259;
-pub const KRYATTR_MAX_LOGIN_ATTEMPTS: CK_ULONG =
-    CKA_VENDOR_DEFINED + KRYATTR_OFFSET + 1;
+pub const KRY_VENDOR_OFFSET: CK_ULONG = CKA_VENDOR_DEFINED + 485259;
 
-pub const KRYERR_OFFSET: CK_ULONG = 485259;
-pub const KRYERR_TOKEN_NOT_INITIALIZED: CK_ULONG =
-    CKR_VENDOR_DEFINED + KRYERR_OFFSET + 1;
+/* Object types */
+pub const KRY_TOKEN_DATA: CK_OBJECT_CLASS = KRY_VENDOR_OFFSET + 1;
```

Comment:
Should we do the `KRO_` for object class?

The current way is slightly non-consistent.

Reply:
Will change